linters/eslint/CHANGELOG.md

```diff
@@ -1,5 +1,7 @@
 # Change Log
 
+## 0.0.0-20241209155954
```

Comment:
Just noting that this file got flagged as having a change, and this version number seems to correspond to a timestamp that I'd assume is a snapshot release. Maybe you don't want this change to be in here?